metadata/metadata.go

```diff
@@ -81,12 +81,12 @@ func Pairs(kv ...string) MD {
 	return md
 }
 
-// Len returns the number of items in md.
+// Len returns the number of items in MD.
```

Comment:
Another question: do you think using the object name "md" by itself is sufficient or should more readable name be used?